src/Core/Services/Implementations/EventService.cs

```diff
@@ -249,7 +257,16 @@ private async Task CreateLogOrganizationUserEventsAsync(IEnumerable<(Organizatio
                 ActingUserId = _currentContext?.UserId,
                 Date = date.GetValueOrDefault(DateTime.UtcNow),
                 SystemUser = systemUser
-            });
+            };
+
+            if (systemUser is EventSystemUser.SCIM)
```

Comment:
Same thing as on the `CreateLogGroupEventAsync` method - set `DeviceType = systemUser.HasValue ? DeviceType.Server : null` in the `EventMessage` class instantiation.